README.md

```diff
@@ -0,0 +1,194 @@
+# redux-mock-store-await-actions
+
+[![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url] [![Greenkeeper badge][greenkeeper-image]][greenkeeper-url] ![Conventional Commits][conventional-commits-url]
+
+[npm-url]:https://npmjs.org/package/redux-mock-store-await-actions
+[npm-image]:http://img.shields.io/npm/v/redux-mock-store-await-actions.svg
+[downloads-image]:http://img.shields.io/npm/dm/redux-mock-store-await-actions.svg
+[travis-url]:https://travis-ci.org/moxystudio/redux-mock-store-await-actions
+[travis-image]:http://img.shields.io/travis/moxystudio/redux-mock-store-await-actions/master.svg
+[codecov-url]:https://codecov.io/gh/moxystudio/redux-mock-store-await-actions
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/redux-mock-store-await-actions/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/redux-mock-store-await-actions
+[david-dm-image]:https://img.shields.io/david/moxystudio/redux-mock-store-await-actions.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/redux-mock-store-await-actions?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/redux-mock-store-await-actions.svg
+[greenkeeper-image]:https://badges.greenkeeper.io/moxystudio/redux-mock-store-await-actions.svg
+[greenkeeper-url]:https://greenkeeper.io
+[conventional-commits-url]:https://img.shields.io/badge/Conventional%20Commits-1.0.0-yellow.svg
+
+> Waits for actions dispatched asynchronously until a timeout expires.
```

Comment:
I think the text you have on the API section is a better summary and explanation than this.
> Returns a Promise which fulfils if all actions are dispatched before the timeout expires, otherwise it is rejected.

Reply:
Indeed, but I think the project description should be short and straight to the point instead, effectively describing just _what_ the module does. The details of _how_ it does, should be strictly reserved to the API section. Furthermore, the project description matches the description in `package.json`.

Reply:
I agree. Then at the very least lets try to make the description less misleading. Right now it sounds like it is the timeout that drives the behaviour when, AFAIK, that's not exactly right.

Maybe something like `Waits for specific actions to be dispatched or until a timeout expires.`?

Reply:
Hmm. The `or` does not sound right, I think, does it?
What about `Waits for specific actions to be dispatched within a time interval?`

Reply:
I think that is still misleading. Most of the times the timeout won't even run, but that sentence makes it sound as if it always does.

What the package does is waiting for some actions to be dispatched and resolve as soon as they do, but it they don't within a specified period of time, then the promise rejects. I think the timeout is a secondary thing here, and we should be clear that the promise will resolve asap. Nothing on that sentence tells me that.